app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -176,40 +176,45 @@ class FollowUsers extends Component {
             </header>
 
             <div data-testid="onboarding-users">
-              {users.map((user) => (
-                <button
-                  data-testid="onboarding-user-button"
-                  type="button"
-                  onClick={() => this.handleClick(user)}
-                  onKeyDown={() => this.handleKeyDown(user)}
-                  className={
-                    selectedUsers.includes(user)
-                      ? 'user content-row selected'
-                      : 'user content-row unselected'
-                  }
-                >
-                  <figure className="user-avatar-container">
-                    <img
-                      className="user-avatar"
-                      src={user.profile_image_url}
-                      alt="profile"
-                    />
-                  </figure>
-                  <div className="user-info">
-                    <h4 className="user-name">{user.name}</h4>
-                    <p className="user-summary">
-                      {he.unescape(user.summary || '')}
-                    </p>
-                  </div>
+              {users.map((user) => {
+                const selected = selectedUsers.includes(user);
+
+                return (
                   <button
-                    data-testid="onboarding-user-following-status"
+                    data-testid="onboarding-user-button"
                     type="button"
-                    className="user-following-status"
+                    onClick={() => this.handleClick(user)}
+                    className={
+                      selected
+                        ? 'user content-row selected'
+                        : 'user content-row unselected'
+                    }
                   >
-                    {selectedUsers.includes(user) ? 'Following' : 'Follow'}
+                    <figure className="user-avatar-container">
+                      <img
+                        className="user-avatar"
+                        src={user.profile_image_url}
+                        alt="profile"
```

Comment:
(non-blocking) it would be another quick accessibility win here to change the `alt` to empty since the profile image is purely decorative

```suggestion
                        alt=""
```